setuptools_py2cfg.py

```diff
@@ -69,6 +69,15 @@ def main(args=None):
     for section, value in sections.items():
         config[section] = value
 
+    # Load the existing setup.cfg if it exists
+    setup_cfg = setuppy_dir / 'setup.cfg'
+    if setup_cfg.exists():
+        setup_cfg_parser = ConfigParser()
```

Comment:
Are you sure that `ConfigParser` is used in setuptools? https://github.com/pypa/setuptools/blob/master/setuptools/config.py#L125

Reply:
Possibly, though I don't now if `ConfigHandler` is part of the public interface.

Is there some situation you can think of where this difference would cause a problem?

Reply:
I just think that if ConfigParser was suitable they wouldn't have created an own parser.

Reply:
If you dig around a bit more, `setuptools` actually *does* use `ConfigParser` for going from `.cfg` to a `config` dictionary. It then has to do a bunch of post-processing on the results to give them real types, because what it gets is basically `Dict[str, Dict[str, str]]`, and it needs to distinguish between lists, booleans, etc, and possibly validate the metadata.

This project doesn't need to do any of that, because it can be assumed that the existing configuration file works fine but missing some values. Once it has the values from `setuptools.setup` and the values from `setup.cfg` (already in a form ready to be written to disk), it can merge them and dump to disk, no need to parse into bools or anything.

Reply:
>setuptools actually does use ConfigParser

thank you for the clarification.